Add RubySur Meetup and Artwork

zhephyn opened the pull request:
<img width="1880" height="1332" alt="CleanShot 2025-07-31 at 13 10 13@2x" src="https://github.com/user-attachments/assets/b5b3440a-d065-4e9e-979a-30ba5e8a68b3" />

Fixes #654

zhephyn committed:
Add RubySur Argentina

marcoroth commented:
Hey @zhephyn, thanks for this pull request! 

It looks like the merge went bad, would you mind rebasing/cherry-picking your commit.

zhephyn committed:
Fix CI fail

zhephyn committed:
Fix lint fail

zhephyn committed:
Merge branch 'main' into rubysur-argentina

zhephyn commented:
@marcoroth All good from my end!

marcoroth commented:
I just realized, this is a meetup, right?

In that case, would you mind combining them into one file like we do with the other meetups? Like: [SF Bay Area Ruby Meetup](https://github.com/rubyevents/rubyevents/blob/main/data/sf-bay-area-ruby/sf-bay-area-ruby-meetup/videos.yml).

zhephyn committed:
Update RubySur Argentina to reflect that it's a monthly meetup

zhephyn committed:
Fix CI fail

zhephyn commented:
@marcoroth So, I updated the changes related to the RubySur argentina meetup. 

However, currently the CI is failing with an error(see below) whereby initially it presented a video id which corresponds to a non existent railsconf 2025 video. The video in question is the talk titled "RailsConf 2025 - The Closing of RailsConf with Chris Oliver and Ufuk [Kayserilioglu"
<img width="1206" height="361" alt="2025-07-30_15-15" src="https://github.com/user-attachments/assets/64de17ac-3686-4127-80e8-43f2df8a27ea" />

I added the video in hopes that this would solve the CI fail but that didn't work. For additional context, this video was published yesterday as the last recording for Rails conf 2025 so it was probably not included in the initial Rails conf 2025 videos as part of that PR. 

The CI fail doesn't make sense because the "no speakers error" shouldn't be raised as speakers are included in the YAML for the added video. Also, for some reason, its processing the video twice instead of once. Also, locally ```rails db:seed``` runs without any issues. 
<img width="1338" height="646" alt="localhost" src="https://github.com/user-attachments/assets/eb78b118-39cb-436f-88c2-2e25eefbff36" />

```ruby
Run bin/rails test test/tasks/db_seed_test.rb
Running 1 tests in a single process (parallelization threshold is 50)
Run options: --seed 52163

# Running:

No speakers for Video ID: ARfpwJ1Ut70
No speakers for Video ID: ARfpwJ1Ut70
No speakers for Video ID: ARfpwJ1Ut70
No speakers for Video ID: ARfpwJ1Ut70
F
Failure:
DbSeedTest#test_db:seed_runs_successfully [test/tasks/db_seed_test.rb:41]:
Expected                      Actual
1 {}                          1 { "ARfpwJ1Ut70" => 2 }
bin/rails test test/tasks/db_seed_test.rb:23
```

marcoroth commented:
Hey @zhephyn, sorry this is an issue with the last commit I pushed to main.

marcoroth commented:
I just fixed it in https://github.com/rubyevents/rubyevents/commit/e645bdb2a18433d2a13d5ec1c333146daaa9a32e

marcoroth committed:
Merge branch 'main' into rubysur-argentina

marcoroth commented on data/railsconf/railsconf-2025/videos.yml:
```diff
@@ -1209,3 +1209,17 @@
   video_provider: "youtube"
   video_id: "xWDhBqNuA7A"
   slides: "https://docs.google.com/presentation/d/1Hz3P5rsdPqUMziviD7Lla78LB0ChrtmEbwPBkEy9dEU/view"
+
+- title: "RailsConf 2025 - The Closing of RailsConf with Chris Oliver and Ufuk Kayserilioglu"
+  raw_title: "RailsConf 2025 - The Closing of RailsConf with Chris Oliver and Ufuk Kayserilioglu"
+  speakers:
+    - Chris Oliver
+    - Ufuk Kayserilioglu
+  date: "2025-07-09"
+  event_name: "RailsConf 2025 (Philadelphia)"
+  published_at: "2025-07-29"
+  announced_at: "TODO"
+  description: |-
+    The end of RailsConf
+  video_provider: "youtube"
+  video_id: "ARfpwJ1Ut70"
```
```suggestion
```

marcoroth committed:
Apply suggestions from code review

marcoroth committed:
Rework content, rename meetup/org, add assets

marcoroth committed:
Remove unused thumbnails

marcoroth committed:
Remove unused thumbnails

marcoroth committed:
Add background color

marcoroth committed:
Lint

marcoroth committed:
Add missing thumbnails
